ngraph/frontend/paddlepaddle/src/op/conv2d_transpose.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "node_context.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs conv2d_transpose(const NodeContext& node_context);
+
+            }
```

Comment:
As I see code style is disabled for PDPD framework. Could you enable code style?

Reply:
@ilyachur Hi I the clang-format check has already been  enabled for PDPD framework.  Enabled at [here](https://github.com/nosovmik/openvino/blob/d04a8667f56974214c8986c7de9e39bae52a0175/ngraph/frontend/paddlepaddle/CMakeLists.txt#L47)

Reply:
@zhangYiIntel Could you double check that it works?
Because I cannot believe that `clang-format` check is enabled, because it should fix code style and add `// namespace op` comment.
Also I saw some empty lines if I remember right, `clang-format` should fix it.

Reply:
Hi @ilyachur , I check that the clang-format check works, but it seems the [clang-format](https://github.com/nosovmik/openvino/blob/pdpd_frontend/ngraph/.clang-format) config doesn't handle that extra empty lines. The config doesn't set `MaxEmptyLinesToKeep`  option.

Reply:
Ok, but it is strange that clang-format doesn't fix namespace.
Because as I see in this config: https://github.com/nosovmik/openvino/blob/pdpd_frontend/ngraph/.clang-format#L54

Clang-format should add the comment with namespace name in this line.

Reply:
Yep, it's strange that the `clang-format` only adds namespace comments in `*.cpp` but not in `*.hpp`

Reply:
Oh, ok, maybe I understand the issue.
Your target doesn't have dependency on this headers.

Could you try to enable `clang-format` `FOR_SOURCES` not `FOR_TARGET`?

I mean change:
```
add_clang_format_target(paddlepaddle_ngraph_frontend_clang FOR_TARGETS paddlepaddle_ngraph_frontend                        EXCLUDE_PATTERNS ${PROTO_SRCS} ${PROTO_HDRS})
```
to something like:
```
add_clang_format_target(paddlepaddle_ngraph_frontend_clang FOR_SOURCES ${ALL_SOURCE_FILES})
```

Reply:
Thanks a lot for your advise. I will try to fix this.  @ilyachur @nosovmik But could we fix this code style issue in another PR ? Since this PR has lots of other functionality features. I am afraid there may be lots of changes if we fix that code style that in this PR.

Reply:
@mvafin , can you try to fix clang for PDPD header files in another PR?